shell/common/shell.cc

```diff
@@ -2023,6 +2031,61 @@ bool Shell::OnServiceProtocolReloadAssetFonts(
   return true;
 }
 
+void Shell::AddView(int64_t view_id, const ViewportMetrics& viewport_metrics) {
+  TRACE_EVENT0("flutter", "Shell::AddView");
+  FML_DCHECK(is_set_up_);
+  FML_DCHECK(task_runners_.GetPlatformTaskRunner()->RunsTasksOnCurrentThread());
+  FML_DCHECK(view_id != kFlutterImplicitViewId)
+      << "Unexpected request to add the implicit view #"
+      << kFlutterImplicitViewId
+      << ". This view should never be added. This call is no-op.";
```

Comment:
The assertion failure should exit the program. Consider:

```suggestion
      << ". This view should never be added.";
```

Reply:
Same comment for `RemoveView` assertion